app/migrate.go

```diff
@@ -0,0 +1,202 @@
+package app
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubeclientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/longhorn/longhorn-manager/datastore"
+	"github.com/longhorn/longhorn-manager/types"
+
+	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+)
+
+const (
+	FlagAll = "all"
+)
+
+func MigrateCmd() cli.Command {
+	return cli.Command{
+		Name: "migrate",
```

Comment:
The name is not specific. Something like `migrate-pre-0.7.0-pv` would be more proper.